changelog/54.bugfix

```diff
@@ -0,0 +1 @@
+The provided filename is now correctly parsed and provided to the job templates.
```

Comment:
better 'Fixed edge-case in input filename parsing.'

It's not like it wasn't parsed correctly before. We just missed a specific edge case.